README.md

```diff
@@ -117,22 +117,26 @@ int main(int argc, char* argv[])
 	// Open the device using the VID, PID,
 	// and optionally the Serial number.
 	handle = hid_open(0x4d8, 0x3f, NULL);
+	if (!handle) {
+		printf("Unable to open device\n");
+ 		return 1;
```

Comment:
```suggestion
		printf("Unable to open device\n");

		hid_exit()
 		return 1;
```

Reply:
Good point, but make the same change in the other test.c 😁